src/main/scala/io/viash/config/resources/Resource.scala

```diff
@@ -96,6 +96,25 @@ trait Resource {
     }
   }
 
+  def resolvedPath: Option[String] = {
```

Comment:
Could you add some inline or function-level docs to describe in which order the path gets resolved? :bow: